cobalt/build/build_info.py

```diff
@@ -57,7 +57,7 @@ def get_build_id_from_commit_count(cwd):
   output = subprocess.check_output(['git', 'rev-list', '--count', 'HEAD'],
                                    cwd=cwd)
   build_id = int(output.strip().decode()) + COMMIT_COUNT_BUILD_ID_OFFSET
-  return build_id
+  return str(build_id)
```

Comment:
Is there a good reason for not making it an `int` in all cases? How is it written in the json?

Reply:
We could make it an int in all cases rather than a string, the important thing is that it's consistent so it doesn't break scripts downstream. Since the current behavior downstream expects a string, this is the one I've gone with.

Reply:
Also string is probably easier since the build_id is concatenated with strings to produce build_version and is added to METADATAs and CL messages.